### DISCUSSION TITLE
MeshCentral doesn't start in docker

### DISCUSSION THREAD

**Discussion by larseberhardt** [amidaware/tacticalrmm#903] [Q&A] [answered]
**Server Info (please complete the following information):**
 - OS: 4.19.0-14-amd64
 - Browser: Chrome
 - RMM Version: v0.10.5 

**Installation Method:**
  - [ ] Standard
  - [X] Docker

**Describe the bug**
I tried to install Tactical RMM with docker. I'd to make some changes on the docker-compose file.
Here is my compose file: https://pastebin.com/9ACC2EdT

I've just changed the subnet and also changed the ip for nginx. 
I run nginx on my server already, so I changed my ports to this:
    ports:
      - "800:80"
      - "4430:443"

I've created a reverse proxy to this ports. I can reach the tactical-frontend but the other containers doesn't start because MeshCentral is waiting for nginx to start…

Here is the pastebin when I've startet docker-compose up -d:
https://pastebin.com/dwMWZhhG

nginx is up and running but meshcentral is waiting for nginx to start. I've no idea why this behaviour happen.

Hopefully someone could help or have a hint.

Thanks and merry Christmas!

**Comment by wh1te909** [comment]
moved to a github discussion since this is unsupported and not an issue
have you checked our unsupported docs? https://wh1te909.github.io/tacticalrmm/unsupported_scripts/

**Reply by larseberhardt** [reply]
Thanks for your reply @wh1te909. I checked the unsupported doc but there nothing listed for a nginx proxy to nginx. I also don't understand why MeshCentral is not starting when the nginx container is started.

**Comment by larseberhardt** [comment, marked as answer]
Hey for all for facing the same issue. It didn't work because I changed the subnet. I just deleted an other subnet, changed the subnet like in docs and then I started the container services. Everything works fine now.

**Reply by wh1te909** [reply]
awesome thanks for posting your findings!
